Mapbox/MapboxMapsFoundation/Camera/CameraManager.swift

```diff
@@ -165,7 +165,7 @@ public class CameraManager {
      - Parameter completion: The completion block to execute after the transition has occurred.
      */
 
-    public func setCamera(to newCamera: CameraOptions,
+    public func setCamera(to camera: CameraOptions,
```

Comment:
@julianrex we spoke about cleaning up these methods. But with this change to only having this single setCamera method. It removes the ability to opt out of optimizing bearing. We should probably add that logic here

Reply:
Based on discussion with @julianrex we'll move forward with this change and revisit adding an option to opt-out of bearing optimization.